cmd/skupper/skupper.go

```diff
@@ -476,6 +481,8 @@ installation that can then be connected to other skupper installations`,
 	cmd.Flags().StringVar(&routerCreateOpts.ConfigSync.CpuLimit, "config-sync-cpu-limit", "", "CPU limit for config-sync pods")
 	cmd.Flags().StringVar(&routerCreateOpts.ConfigSync.MemoryLimit, "config-sync-memory-limit", "", "Memory limit for config-sync pods")
 
+	cmd.Flags().IntVar(&LoadBalancerTimeout, "timeout", 120, "Configurable timeout in seconds for the ingress loadbalancer option")
```

Comment:
instead of using the literal 120, use the constant DefaultTimeout

Reply:
Right, thanks!